playground-next13-live-editing/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "next13-live-editing",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "demo": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@storyblok/react": "^0.0.1",
+    "eslint": "8.35.0",
+    "eslint-config-next": "13.2.3",
+    "next": "13.2.3",
```

Comment:
Think about upgrade to the latest version, App directory is stable, so you won't have any surprise.